x/rollup/keeper/deposits.go

```diff
@@ -21,6 +29,7 @@ import (
 // Persisted data conforms to optimism specs on L1 attributes:
 // https://github.com/ethereum-optimism/optimism/blob/develop/specs/deposits.md#l1-attributes-predeployed-contract
 func (k *Keeper) setL1BlockInfo(ctx sdk.Context, info derive.L1BlockInfo) error { //nolint:gocritic
+	// TODO: use binary marshaling instead of json
 	infoBytes, err := json.Marshal(info)
```

Comment:
Question to reviewers - Do you think it's worth bringing up an issue for this?

Reply:
Why is it an issue in the first place? If there is a benefit for binary (size?), let's add a good first issue.

Reply:
Yeah mainly just size and performance, we could use protobufs here instead. Sounds good, I'll raise an issue for this

Reply:
https://github.com/polymerdao/monomer/issues/233